From d1fa707f928cdd6036dda0728d18e4a456e54eba Mon Sep 17 00:00:00 2001
From: "Subhransu S. Prusty" <subhransu.sekhar.prusty@xilinx.com>
Date: Thu, 2 May 2019 01:54:23 -0700
Subject: [PATCH 0632/1852] dt-bindings: media: xilinx: Add optional external
 reset for core reset

commit 0875a996856c8bb7b029f3a5212a18008c9d5c4f from
https://github.com/Xilinx/linux-xlnx.git

Add an optional external gpio reset option for video_aresetn. This is
required to reset the MIPI CSI subsystem gracefully in SLBF condition.

Signed-off-by: Subhransu S. Prusty <subhransu.sekhar.prusty@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt          | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
index 20460744b4c6..2e10719fc8b0 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
@@ -69,6 +69,8 @@ Optional Properties
 
 - xlnx,en-vcx: When present, the max number of virtual channels can be 16 else 4.
 
+- reset-gpios: Optional specifier for a GPIO that asserts video_aresetn.
+
 Example:
 
 	csiss_1: csiss@a0020000 {
-- 
2.31.1

